ocs_ci/ocs/ocs_upgrade.py

```diff
@@ -644,14 +644,26 @@ def ocs_odf_upgrade_ui():
     logger.info(
```

Comment:
Please move this logger just before this line so as to maintain the code continuity. 
```
val_obj.do_click(
        upgrade_ocs.validation_loc["storage-system-on-installed-operators"]
    )
```